opm/output/eclipse/Summary.cpp

```diff
@@ -800,13 +824,19 @@ inline quantity rate( const fn_args& args ) {
         sum *= -1.0;
     }
 
+    const auto& gsatprod = args.schedule[args.sim_step].gsatprod.get();
+    if (!injection && gsatprod.size() > 0 && !args.group_name.empty()) {
+        sum += accum_groups(phase, args.schedule, args.sim_step, args.group_name);
+    }
```

Comment:
I'd like to see a comment here for why this is safe.  In particular, this block relies on `group_name` being empty for well-level quantities such as `WOPR` and `WGPT`.